leaf-ui/js/onFunctions.js

```diff
@@ -411,39 +418,46 @@ function reassignIntentionIDs() {
 /**
  * Helper function for switching to Analysis view.
  */
-var inAnalysis = false;
 function switchToAnalysisMode() {
     setInteraction(false);
-    inAnalysis = true;
 	reassignIntentionIDs();
-	
+
 	// Clear the right panel
 	clearInspector();
 	
 	removeHighlight();
     
-    $('#config').append(configInspector.el);
+    $('#configID').append(configInspector.el);
     configInspector.render();
-	$('#stencil').css("display", "none");
+
+    // disappear
```

Comment:
Remember my previous review where I mentioned correct comment format is // First word capitalized